internal/ingress/controller/store/store.go

```diff
@@ -301,6 +310,51 @@ func (s k8sStore) GetServiceEndpoints(key string) (*corev1.Endpoints, error) {
 	return s.listers.Endpoint.ByKey(key)
 }
 
+func (s *k8sStore) annotationToNodeLabelObj(nodeLabelAnnotation string) *[]NodeLabel {
+	var nodeLabels []NodeLabel
+	cleaned := strings.Replace(nodeLabelAnnotation, " ", "", -1)
```

Comment:
1. The type matches our stringMap type in [our doc](https://kubernetes-sigs.github.io/aws-alb-ingress-controller/guide/ingress/annotation/).
I'd like to use existing logic to maintain same behavior among other annotations:
    1. support  for customize annotation prefix via [--alb-name-prefix](https://github.com/kubernetes-sigs/aws-alb-ingress-controller/blob/master/internal/ingress/controller/config/config.go#L75), which allows you to use another prefix instead of 'alb.ingress.kubernetes.io'
    2. maintain same behavior on how to parse annotations. e.g. 'a=b c=d' will be parsed differently under current implementation.

I'd suggest add an method with syntax `func LoadStringMapAnnotation(annotation string, value *map[string]string, annotations ...map[string]string) bool {` [here](https://github.com/kubernetes-sigs/aws-alb-ingress-controller/blob/master/internal/ingress/annotations/annotations.go)
Also, see [sample code](https://github.com/kubernetes-sigs/aws-alb-ingress-controller/blob/master/internal/ingress/annotations/tags/main.go#L41) here for implementation same behavior.

2. The currently solution requires the annotation to be set on ingress only. But ideally this should be allows to set on either ingress or service and the annotation on service takes priority. (since the backend(targetGroup) differs by service).
See sample code [here](https://github.com/kubernetes-sigs/aws-alb-ingress-controller/blob/b3de77759486af1bfb9d75960942723544437c06/internal/ingress/auth/auth.go#L101) on how to handle it